src/m1_facilitator/handlers/DeclaredDepositIntentsHandler.ts

```diff
@@ -169,4 +178,16 @@ export default class DeclaredDepositIntentsHandler extends ContractEntityHandler
       Logger.debug(`Deposit intent ${depositIntent} saved.`);
     }
   }
+
+  /**
+   * Checks if value token address needs to be facilitated.
+   *
+   * @param valueTokenAddress Value token address.
+   */
+  private isFacilitateToken(valueTokenAddress: string): boolean {
```

Comment:
`isFacilitateToken` is not providing the context on what it is trying to do, should we rename it? something like `shouldFacilitateForToken` or `isTokenSupported`.

Reply:
renamed to `isTokenSupported`